### ISSUE TITLE
Specify the color space (sRGB) in all non-vector images

### ISSUE THREAD

**Issue by InessaPawson** [numpy/numpy.org#399]
Specify the color space (sRGB) in all numpy.org non-vector images.

**Comment by Shubham-Vishwakarma** [comment]
@InessaPawson 
Is this issue still open?

If yes
Will you please provide information on this

**Comment by InessaPawson** [comment]
@Shubham-Vishwakarma Yes, it is open. Go for it if you are interested!
This task requires examining all the non-vector images in the numpy.org repo and making sure that the color space is set to RGB.

**Comment by Shubham-Vishwakarma** [comment]
@InessaPawson 

I am new to HTML/CSS stuff

Will you please confirm I need perform similar action as done in https://github.com/aerikpawson/numpy.org/commit/85e321c7e6ca5ae4c80e74014465aed616af2dcb

**Comment by Shubham-Vishwakarma** [comment]
https://github.com/numpy/numpy.org/pull/404

This seems to be a better reference

**Comment by Shubham-Vishwakarma** [comment]
@InessaPawson 
When you say non-vector images, do you mean .png/.jpg???

**Comment by InessaPawson** [comment]
> @InessaPawson
> When you say non-vector images, do you mean .png/.jpg???

@Shubham-Vishwakarma  Correct. This task requires some knowledge of graphic design software. Adobe Photoshop, GIMP or their alternatives would be best for this.

**Comment by InessaPawson** [comment]
> #404
> This seems to be a better reference
@Shubham-Vishwakarma Yes, it’s more relevant. But you need to keep in mind that you will be working with pixels, not mathematical formulas like in the case of vector images.

**Comment by Mann-tech13** [comment]
@InessaPawson  Can I work on this issue?

**Comment by rgommers** [comment]
@Mann-tech13 please do. Any issue you think you can work on in this repo is fair game; we don't pre-assign issues to people normally.

**Comment by new-vivek11** [comment]
I may start work on this ... Will give update if I start working on it.
